packages/reports/addon/components/visualization-toggle.hbs

```diff
@@ -0,0 +1,18 @@
+{{!-- Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<div class="visualization-toggle" ...attributes>
+  <DenaliToggle
+    @options={{@validVisualizations}}
+    @activeOption={{this.activeVisualization}}
+    @onChange={{fn @onVisualizationTypeUpdate this.clickedVisualization}}
```

Comment:
We should call `@onVisualizationTypeUpdate`  instead of a local property. You might have to do something like
```suggestion
    @onChange={{pipe (pick "name") @onVisualizationTypeUpdate}}
```